### ISSUE TITLE
Get Transactions For Account not returning any transactions

### ISSUE THREAD

**Issue by john-s-exp** [ConsumerDataStandardsAustralia/java-artefacts#116]
Hi Team, 
No transactions are returned for the data-holder /banking/accounts/{accountId}/transactions after several tests. Using accountId 3F7C0D90-05CE-4225-8211-3B4F16AEAEB9 which matches Transaction1.json doesn't work as well. Can you provide a working example/request which returns several transactions?

**Comment by vadkor** [comment]
Hi @john-s-exp. Actually, _transaction1.json_ was pointing to another account, which wasn't in any of the account payloads. I changed it to yours. There was also a small bug in `data-holder` that treated the `oldestTime` and `newestTime` parameters as mandatory. Fixed that and added the test data to the BankingAccountIT.getTransactions() test.
You are welcome to add more test cases and test data. We encourage the community to fork the repo and create pull requests to the *dev* branch of this repository with fixes and contributions.
